akka-docs/src/main/paradox/typed/guide/modules.md

```diff
@@ -180,7 +180,7 @@ Persistence tackles the following challenges:
   symbol1=AkkaVersion
   value1="$akka.version$"
   group=com.typesafe.akka
-  artifact=akka-cluster-typed_$scala.binary.version$
+  artifact=akka-distributed-data_$scala.binary.version$
```

Comment:
Since the new (typed) distributed data API's are now part of akka-cluster-typed (https://github.com/akka/akka/tree/master/akka-cluster-typed/src/main/scala/akka/cluster/ddata/typed), this might actually be the right reference?

We should add a link from here to the actual docs on this feature, I think, though.

Reply:
@raboof  If distributed data is now part of akka-cluster-typed. Can we move distributed data description as a part of cluster-typed.

![image](https://user-images.githubusercontent.com/1023936/94459771-722ac700-01d5-11eb-86ce-08c7bde7a1d5.png)

![image](https://user-images.githubusercontent.com/1023936/94459857-8a024b00-01d5-11eb-9533-0d55c7ab84c3.png)

Reply:
I think we want to keep Distributed data as a separate section of the docs even if the same module provides it.

Reply:
Okay, thanks.! I'll close this PR.